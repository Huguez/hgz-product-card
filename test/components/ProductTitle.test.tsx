import React from "react"
import renderer from "react-test-renderer"
import { ProductTitle, ProductCard } from '../../src/components';
import { product } from '../data/data';

describe( 'ProductTitle', () => {
  
   it('debe de mostrarse econ el titulo personalizado', () => {
      const wrapper = renderer.create( <ProductTitle title="Hola wawa" /> )
      expect( wrapper.toJSON() ).toMatchSnapshot()
   } );

   test( "debe de mostrar el componenete con el nombre del producto", () => {
      const wrapper = renderer.create( 
         <ProductCard product={ product }>
            {
               ()=>(
                  <ProductTitle /> 
               )
            }
         </ProductCard>
      )
      expect( wrapper.toJSON() ).toMatchSnapshot()
   } );

} );
