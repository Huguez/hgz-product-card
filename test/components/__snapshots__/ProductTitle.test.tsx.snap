// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTitle debe de mostrar el componenete con el nombre del producto 1`] = `
<div
  className="undefined  productCard "
>
  <span
    className="undefined  productDescription"
  >
    Coffe Mug - Card
  </span>
</div>
`;

exports[`ProductTitle debe de mostrarse econ el titulo personalizado 1`] = `
<span
  className="undefined  productDescription"
>
  Hola wawa
</span>
`;
